config/settings/prod.py

```diff
@@ -10,13 +10,34 @@
 
 from .base import *
 
+required_env_vars = [
+    "HOST_CONTAINER_NAME",
+    "EXTERNAL_URL",
+    "CANDIG_INTERNAL_DOMAIN",
+    "AGGREGATE_COUNT_THRESHOLD",
+    "OPA_URL",
+    "POSTGRES_DATABASE",
+    "POSTGRES_USER",
+    "POSTGRES_PASSWORD_FILE",
+    "POSTGRES_HOST",
+    "POSTGRES_PORT",
+    "REDIS_PASSWORD_FILE",
+]
+
+missing_vars = [var for var in required_env_vars if not os.getenv(var)]
```

Comment:
This file switches between using `os.getenv()` and `os.environ[]` in several places -- I'd recommend sticking to one or the other (as far as I understand it, one's a wrapper of the other)

Reply:
There is a slight difference, and it's on purpose. `os.getenv` returns None if the variable is not found, allowing me to catch it and throw an exception.

Why not use `os.getenv` below? If it returns None for a missing variable, it can cause the bug we encountered before. If I (or someone else) tries to get a new variable but forgets to add it to required_vars, it will crash instead of causing a subtle bug, making it easier to catch.

Reply:
I see -- perhaps it would be best if lines 34-36 used `os.getenv` then, rather than `os.environ.get()`? For consistency